### DISCUSSION TITLE
How do I convert a trained model to onnx

### DISCUSSION THREAD

**Discussion by luoyq6** [open-edge-platform/anomalib#1977] [Q&A]
### What is the motivation for this task?

How do I convert a trained model to onnx

### Describe the solution you'd like

How do I convert a trained model to onnx

**Comment by lathashree01** [comment]
Hi, 
There is a function under anomalib.deploy to convert trained model to onnx.

https://anomalib.readthedocs.io/en/v1.0.1/markdown/guides/reference/deploy/index.html#anomalib.deploy.export_to_onnx

**Comment by samet-akcay** [comment]
You could also refer to this link
https://anomalib.readthedocs.io/en/latest/markdown/guides/reference/engine/index.html#anomalib.engine.Engine.export

Also, once this PR https://github.com/openvinotoolkit/anomalib/pull/1803 is merged, you will be able to do `model.export(...)`

**Comment by luoyq6** [comment]
Hi, I have converted ONNX successfully, but she can only reason on the cpu, and will fail on the GPU build @samet-akcay
